facilitator/src/bin/facilitator.rs

```diff
@@ -793,6 +841,76 @@ fn aggregate(sub_matches: &ArgMatches) -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+fn lint_manifest(sub_matches: &ArgMatches) -> Result<(), anyhow::Error> {
+    let manifest_base_url = sub_matches.value_of("manifest-base-url");
+    let manifest_file_reader = match sub_matches.value_of("manifest-path") {
+        Some(path) => Some(File::open(path).context("failed to open manifest file")?),
+        None => None,
+    };
+
+    let manifest_kind = ManifestKind::from_str(
+        sub_matches
+            .value_of("manifest-kind")
+            .context("manifest-kind is required")?,
+    )?;
+
+    match manifest_kind {
+        ManifestKind::IngestorGlobal => {
+            let manifest = if let Some(base_url) = manifest_base_url {
+                IngestionServerGlobalManifest::from_https(base_url)?
+            } else if let Some(reader) = manifest_file_reader {
+                IngestionServerGlobalManifest::from_reader(reader)?
+            } else {
+                return Err(anyhow!(
+                    "one of manifest-base-url or manifest-path is required"
+                ));
+            };
+            println!("Valid: {:?}\n{:#?}", manifest.validate(), manifest);
+        }
+        ManifestKind::DataShareProcessorGlobal => {
+            let manifest = if let Some(base_url) = manifest_base_url {
+                DataShareProcessorGlobalManifest::from_https(base_url)?
+            } else if let Some(reader) = manifest_file_reader {
+                DataShareProcessorGlobalManifest::from_reader(reader)?
+            } else {
+                return Err(anyhow!(
+                    "one of manifest-base-url or manifest-path is required"
+                ));
+            };
+            println!("{:#?}", manifest);
```

Comment:
There's some variation among what gets printed for the different cases; is that intentional?

Reply:
Yes: there's no validate method on this one kind of manifest.